make/include/defaults

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+set -o errexit -o nounset
+
+: "${GIT_ROOT:=$(git rev-parse --show-toplevel)}"
+source "${GIT_ROOT}/.envrc"
+
+: "${NAMESPACE:=cf}"
+: "${CF_NAMESPACE:=${NAMESPACE}}"
+: "${CF_CHART:="${GIT_ROOT}/output/helm"}"
+: "${DOMAIN:=cf-dev.io}"
+: "${UAA_NAMESPACE:=uaa}"
+: "${UAA_HOST:=uaa.${DOMAIN}}"
+: "${UAA_PORT:=2793}"
+: "${UAA_CLIENT_ID:=cf}"
```

Comment:
And `UAA_CLIENT_ID` has to be `cf`, due to the [hard-coded client id](https://github.com/SUSE/scf/blob/04f0c59c6d55151d2a440c11e9508d16f424d668/container-host-files/etc/scf/config/opinions-common.yml#L431)?